packages/bff/src/services/catalogService.ts

```diff
@@ -122,5 +123,41 @@ export function catalogServiceBuilder(
 
       return response;
     },
+    createEService: async (
+      eServiceSeed: bffApi.EServiceSeed,
+      { headers }: WithLogger<BffAppContext>
+    ): Promise<CreatedResource> => {
+      const { id } = await catalogProcessClient.createEService(eServiceSeed, {
+        headers,
+      });
+      return { id };
+    },
+    updateEServiceById: async (
+      eServiceId: string,
```

Comment:
[Very minor]
I suggest to using the same naming convention for parameter as `eserviceId` and branded type
```suggestion
      eserviceId: EServiceId,
```